src/components/Articles.jsx

```diff
@@ -1,34 +1,46 @@
-import {useParams} from 'react-router-dom';
+import {useParams,Link} from 'react-router-dom';
 import {useState, useEffect} from 'react';
 import axios from 'axios';
+import Pagination from './Pagination';
 
 function Articles() {
```

Comment:
again this whole file should not have been amended in a branch dedicated to task 5